_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
```

Comment:
I wasn't aware of this file being necessary at first, however, creating `$app_root/tmp/secret_token.rb` and setting it to the same value across all my Foreman servers did solve my issue: https://groups.google.com/forum/?fromgroups#!topic/foreman-users/OeOvX_IBkcw . I've had mixed results, I do have multiple dev instances without this file that work (work = can log in w/o Invalid Authenticity Token warning), yet my bigger prod instance doesn't work without this file. If you'd like, we can chat about this on IRC a bit, but I can tell you that `$app_root/config/initializers/local_secret_token.rb` was present and set the same across all servers before I tried secret_token.rb.